app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx

```diff
@@ -4,16 +4,14 @@
 
 import React from "react";
 import { RouteComponentProps } from "@reach/router";
-import AppLayoutWithSidebar from "../AppLayoutWithSidebar";
+import PageEditContainer from "../PageEditContainer";
 
 type PageRequestReviewProps = {} & RouteComponentProps;
 
 const PageRequestReview: React.FunctionComponent<PageRequestReviewProps> = () => (
-  <AppLayoutWithSidebar>
-    <section data-testid="PageRequestReview">
-      <h1>PageRequestReview</h1>
-    </section>
-  </AppLayoutWithSidebar>
+  <PageEditContainer title="Review & Launch" testId="PageRequestReview">
```

Comment:
I like taking out the `isLoading` and `notFound` if check redundancies and this PR overall, but I don't believe we will want to use `PageEditContainer` on the "Review & Launch" page.

Since that page is a "review" page and not a page in which you can edit an experiment, I don't believe we want `HeaderEditExperiment` on there, plus we'll need some kind of polling. I like this component for `PageEditOverview`, `PageEditBranches`, and later `PageEditAudience` and `PageEditMetrics` though.

Reply:
Oh? I was just going off of what the Figma doc showed (it has the header bit)

Reply:
@jodyheavener oh nope you're right, it is present with the most recent comps. Sorry, Figma is ever-evolving, I still pictured one day this turning into an "edit" form in which case I didn't think we'd want to allow that on the "review" page. I bet we can finesse polling in there with some option, otherwise whomever works on that can take the request review page out of this container so seems good to me.